### DISCUSSION TITLE
override default type mapping  SQLite DateTime

### DISCUSSION THREAD

**Discussion by rhscjohn-dev** [prisma/prisma#9016] [Q&A]
According to the docs,  DateTime  has a default type mapping of NUMERIC.  

SQLite is capable of storing dates and times as TEXT, REAL, or INTEGER.  My existing database stores all dateTime  fields  as strings(TEXT).

Is there a way in the Model Schema  to change the default type from NUMERIC to TEXT?

Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
